analysis/create_project_actions.R

```diff
@@ -165,19 +165,40 @@ clean_data <- function(cohort, describe = describe) {
 
 # Create function for table1 --------------------------------------------
 
-table1 <- function(cohort, ages = "18;40;60;80") {
+table1 <- function(cohort, ages = "18;40;60;80", preex = "All") {
   splice(
-    comment(glue("Table 1 - {cohort}")),
-    action(
-      name = glue("table1_{cohort}"),
-      run = "r:latest analysis/table1/table1.R",
-      arguments = c(c(cohort), c(ages)),
-      needs = list(glue("clean_data_{cohort}")),
-      moderately_sensitive = list(
-        table1 = glue("output/table1/table1_{cohort}.csv"),
-        table1_midpoint6 = glue("output/table1/table1_{cohort}_midpoint6.csv")
+    if (preex == "All") {
+      comment(glue("Table 1 - {cohort}"))
+    } else {
+      comment(glue("Table 1 - {cohort} - {preex}"))
```

Comment:
At the moment, the names print as 'Table 1 - vax - FALSE' (for example) and it is unclear what the TRUE/FALSE refers to. Try this instead:
```suggestion
      comment(glue("Table 1 - {cohort} - preex_{preex}"))
```